src/devices/sound/cf61909.h

```diff
@@ -0,0 +1,62 @@
+// license: BSD-3-Clause
+// copyright-holders: Devin Acker
+
+#ifndef MAME_SOUND_CF61909_H
+#define MAME_SOUND_CF61909_H
+
+#include "machine/clock.h"
+#include "dirom.h"
+
+#include <array>
+
+#pragma once
```

Comment:
Please keep the include guard parts (the preprocessor stuff and the `#pragma`) together.